lib/Echidna/Test.hs

```diff
@@ -36,14 +36,16 @@ data CallRes = ResFalse | ResTrue | ResRevert | ResOther deriving (Eq, Show)
 classifyRes :: VMResult -> CallRes
 classifyRes (VMSuccess b) | b == encodeAbiValue (AbiBool True)  = ResTrue
                           | b == encodeAbiValue (AbiBool False) = ResFalse
+                          | otherwise                           = error "invalid result"
```

Comment:
We've been classifying these as `ResOther` so far. Do we really want to start throwing instead?

Reply:
I believe this will never happen.

Reply:
I think it could, if someone writes a test like, really wrong (with inline asm for instance)

Reply:
Fair enough. It returns `ResOther` now.